helix-rest/src/test/java/org/apache/helix/rest/metadatastore/accessor/TestZkRoutingDataWriter.java

```diff
@@ -40,15 +41,35 @@
   private static final String DUMMY_SHARDING_KEY = "SHARDING_KEY";
   private MetadataStoreRoutingDataWriter _zkRoutingDataWriter;
 
+  // MockWriter is used for testing request forwarding features in non-leader situations
+  class MockWriter extends ZkRoutingDataWriter {
```

Comment:
Nit: MockZkRoutingDataWriter so there's no ambiguity?

Reply:
Ditto below. This mock writer is for unit testing `ZkRoutingDataWriter`, it's not the one for integration testing. Calling this `MockZkRoutingDataWriter` actually makes it ambiguous because that would create two classes of `MockZkRoutingDataWriter`.

Reply:
OK